static/js/components/PersonalForm.js

```diff
@@ -39,6 +40,28 @@ export default class PersonalForm extends ProfileFormFields {
     ui:                     UIState,
   };
 
+  renderRomanizedFields = () => {
+    const { profile } =  this.props;
+    if (showRomanizedFields(profile)) {
+      return (
+        <Cell col={12}>
+          <section className="romanized-name">
+            <h3>Plese enter your name in Latin characters</h3>
```

Comment:
Typo: "Please", not "Plese". Also, I'm not sure that everyone will know what "Latin characters" are. Maybe we should provide some examples, and an explanation for why we are asking this. I would recommend:

> Please enter your name using only Latin characters. For example, convert "è" to "e", "ä" to "a", "ñ" to "n", and so on. This is necessary because some of our systems can only handle Latin characters.